tools/fakeDeposit.ts

```diff
@@ -0,0 +1,50 @@
+import { SendMail } from '@thecointech/email';
+import { randomBytes } from "crypto";
+import { DateTime } from 'luxon';
+import yargs from 'yargs';
+import { log } from '@thecointech/logging';
+
+async function fakeDepositEmail(address: string, amount: number) {
+
+  const randomId = randomBytes(4).toString('hex');
+  const p1 = randomBytes(32).toString('hex');
+  const expiry = DateTime.now().plus({month: 1})
+  log.debug(`Sending $${amount} to ${address}`);
+
+  SendMail(
+    "[REDIRECT:] INTERAC e-Transfer: (fake deposit) has sent you money.",
+    `
+Hi Coin,
```

Comment:
Utility script to send a fake deposit to our testing processor.

This allows complete E2E testing of the prodtest